### ISSUE TITLE
Missing icons for shortcuts on front page

### ISSUE THREAD

**Issue by redoomed1** [privacyguides/discourse-brand#4]
Below are screenshots of the issue that appear in multiple forum themes.

**WCAG Dark**

<img width="538" alt="Image" src="https://github.com/user-attachments/assets/5b05cc14-9f7a-4751-bf35-25ecaa207139" />

**Default Dark**

<img width="538" alt="Image" src="https://github.com/user-attachments/assets/cfc96207-dc4c-467f-aa2b-58995244effc" />

**Default Light**

<img width="538" alt="Image" src="https://github.com/user-attachments/assets/d525d568-79c3-49d9-a0fd-e0a838a429de" />

**WCAG Light**

<img width="538" alt="Image" src="https://github.com/user-attachments/assets/c6ce9437-b235-4288-a0c9-3ce067841665" />
